### ISSUE TITLE
Unable to add sprites past ID 483

### ISSUE THREAD

**Issue by Ryguy0777** [RoadrunnerWMC/Reggie-Updated#27]
trying to add a sprite to spritedata.xml with ID >483 causes this upon running
![image](https://github.com/NSMBW-Community/Reggie-Updated/assets/118482718/3e3ec337-7bb0-4ae3-9866-4d06ee020af9)
it seems that the sprite limit is hardcoded at 483. This issue is not present in Reggie! Next.

**Comment by Meatball132** [comment]
Yep, it's a hardcoded magic number near the top of LoadSpriteData()

This has become a problem for me as well, so I concur with this issue report.
